openstates/nm/bills.py

```diff
@@ -97,8 +97,10 @@ class NMBillScraper(BillScraper):
     categorizer = Categorizer()
 
     def _init_mdb(self, session):
+        print session[2:]
         ftp_base = 'ftp://www.nmlegis.gov/other/'
-        fname = 'LegInfo{}'.format(session[2:])
+        #session slug here?
+        fname = 'LegInfo{}'.format('16 Special')
```

Comment:
This hack is bad, and i should feel bad. 

TODO: Figure out the right way to format this that works across sessions.